addons/discuss-button/r2.js

```diff
@@ -5,8 +5,9 @@ export default async function ({ addon }) {
   const add = async () => {
     if (!span && (await addon.auth.fetchIsLoggedIn())) {
       const username = await addon.auth.fetchUsername();
-      const container = document.querySelector(".dropdown");
-      const dropdown = container.querySelector(".dropdown-menu .user-nav");
+      const container = await addon.tab.waitForElement(".dropdown");
+      const dropdown = await addon.tab.waitForElement(".dropdown-menu .user-nav");
+      (await addon.tab.waitForElement(".user-icon")).style = "margin-right: 0px;";
```

Comment:
Is there a reason for the extra set of parenthacies?

Reply:
I think the browser would read it as
```js
(await addon.tab.waitForElement(".user-icon").style) = "margin-right: 0px;";
```
without them.